docs/catalog/cloud/Scale Kubernetes workloads based on events.md

```diff
@@ -0,0 +1,38 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- serverless
+- kubernetes
+- role:software-engineer
+- role:cloud-engineer
+- size:medium
+---
+
+# Scale Kubernetes workloads based on events
+
+## Description
+By default, Kubernetes scales workloads based on CPU and RAM utilization, in practice however, it is difficult to define the right thresholds for scaling based on CPU and RAM.
+
+Scaling your workload based on relevant business metrics such as HTTP requests, queue length, and cloud events can help reduce its resource utilization, hence its carbon emissions.
+
+## Solution
+
+Use [KEDA](https://keda.sh) when building event-driven kubernetes workloads, to allow scaling down to zero when there is no demand.
```

Comment:
I'm still not clear what the proposed solution is here. Is it to use metrics other than CPU and RAM utilization to scale your services? How is using 1000 HTTP requests or queue length > 100 more emissions efficient than CPU utilization?